tests/server/test_server_calling_routes.py

```diff
@@ -486,6 +501,58 @@ def test_start_recording__returns_error_code_and_message_if_barcode_is_invalid(
     assert response.status.endswith(expected_error_message) is True
 
 
+@pytest.mark.parametrize(
+    "test_barcode,test_description",
+    [
+        (
+            "MA210440001",
+            "allows year to be 21",
+        ),
+        (
+            "MA190440001",
+            "allows year to be 19",
+        ),
+    ],
+)
+def test_start_recording__allows_years_other_than_20_in_barcode(
+    test_barcode,
+    test_description,
+    client_and_server_thread_and_shared_values,
+    generic_start_recording_info_in_shared_dict,
+):
+    test_client, _, _ = client_and_server_thread_and_shared_values
+    response = test_client.get(f"/start_recording?barcode={test_barcode}")
+    assert response.status_code == 200
+
+
+@pytest.mark.parametrize(
+    "test_barcode,test_description",
+    [
+        (
+            "MA200440001",
+            "allows header 'MA'",
+        ),
+        (
+            "ME200440001",
+            "allows header 'ME'",
+        ),
+        (
+            "MB200440001",
+            "allows header 'MB'",
+        ),
+    ],
+)
+def test_start_recording__allows_years_correct_barcode_headers(
```

Comment:
Looks like this test name accidentally got messed up. I think it was meant to be: `test_start_recording__allows_correct_barcode_headers`

Reply:
fixed.  yes, there's a few places where "else is never true" and things like that in the refactor that I am putting off until the beta1.7 release. there aren't any actual lines that aren't covered.